msu/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -13,41 +13,28 @@
 	local general_querySkillTooltipData = o.general_querySkillTooltipData;
 	o.general_querySkillTooltipData = function( _entityId, _skillId )
 	{
-		local entity = ::Tactical.getEntityByID(_entityId);
+		local ret = general_querySkillTooltipData(_entityId, _skillId);
 
-		if (entity != null)
+		if (ret != null)
 		{
-			local skill = entity.getSkills().getSkillByID(_skillId);
-
-			if (skill != null)
-			{
-				local tooltip = skill.getTooltip();
-				skill.getContainer().onQueryTooltip(skill, tooltip);
-				return tooltip;
-			}
+			local skill = ::Tactical.getEntityByID(_entityId).getSkills().getSkillByID(_skillId);
+			skill.getContainer().onQueryTooltip(skill, ret);
```

Comment:
Probably just add the same checks as the base function (entity != null && skill != null). It's technically not necessary as the base function only returns non-null if those conditions aren't met, but there might be mods that redirect things in that case.

Reply:
Actually it's probably not important, we should come first

Reply:
I don't think anyone should change that function to return a non-null value if either entity or skill is null because that's the whole point of the function.